server/msteams/client.go

```diff
@@ -1119,6 +1119,9 @@ func (tc *ClientImpl) GetReply(teamID, channelID, messageID, replyID string) (*M
 }
 
 func (tc *ClientImpl) GetUserAvatar(userID string) ([]byte, error) {
+	if tc.client == nil {
+		return nil, errors.New("client not initialized")
+	}
```

Comment:
This seems unrelated to these changes -- should we include it in another PR?

Reply:
Let's create a ticket for ensure the plugin doesn't crashes if the plugin is not properly configured.

Reply:
removed from here